src/components/layout/header.tsx

```diff
@@ -117,21 +100,58 @@ const Header = ({ token }: TokenProps) => {
     <HeaderContainer>
       {(token || tokenData) && (
         <>
-          <HeaderBackImageWrap>
-            <KeyboardBackspaceIcon onClick={handleBackClick} />
-          </HeaderBackImageWrap>
-
           <HeaderIconWrap>
             <NavbarIcons>
-              <AccountCircleIcon onClick={handleClickMypage} />
-              <ExitToAppIcon onClick={handleClickLogout} />
+              <AccountCircleIcon ref={anchorRef} onClick={handleOpenToggle} />
+              {/*<AccountCircleIcon onClick={handleClickMypage} />*/}
```

Comment:
mui에서 IconButton이라는 컴폰넌트 사용하면 좋을 것 같습니다. 그러면 허버 효과도 있고 버튼 역할 설정이 되어있어 키보드 네비게이션도 가능해집니다.

```suggestion
<IconButton ref={anchorRef} onClick={handleOpenToggle}>
    <AccountCircleIcon />
</IconButton>
```

Reply:
호우 이런게 있었군요 좋습니다ㅎㅎ